CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  
 ### Fixed
 
+ - #1871, OpenAPI fix: Show missing default values for String types and identify Array types as "array" instead of "string" - @laurenceisla
```

Comment:
```suggestion
 - #1871, Fix OpenAPI missing default values for String types and identify Array types as "array" instead of "string" - @laurenceisla
```